### PR TITLE
OBSDOCS-903: Issue in file monitoring/troubleshooting-monitoring-issu…

### PR DESCRIPTION
**Pull request by eromanova97**
Version(s): `enterprise-4.12` and later

Issue: [OBSDOCS-903](https://issues.redhat.com/browse/OBSDOCS-903)

Link to docs preview: [Investigating why user-defined project metrics are unavailable](https://72973--ocpdocs-pr.netlify.app/openshift-enterprise/latest/monitoring/troubleshooting-monitoring-issues#investigating-why-user-defined-metrics-are-unavailable_troubleshooting-monitoring-issues)

QE review:
- [x] QE has approved this change.

Peer review:
- [x] Peer review done

**Additional information:**

### CODE REVIEW

File: modules/monitoring-investigating-why-user-defined-metrics-are-unavailable.adoc [hunk 1]
```diff
@@ -39,18 +39,21 @@ $ oc -n ns1 get service prometheus-example-app -o yaml
     app: prometheus-example-app
 ----
 +
-.. Check that the `matchLabels` `app` label in the `ServiceMonitor` resource configuration matches the label output in the preceding step:
+.. Check that the `matchLabels.app` label in the `ServiceMonitor` resource configuration matches the label output in the preceding step. The following example queries the `prometheus-example-monitor` service in the `ns1` project:
```
Comment (juzhao) [inline modules/monitoring-investigating-why-user-defined-metrics-are-unavailable.adoc:42]:
```suggestion
.. Check that the `matchLabels.app` label in the `ServiceMonitor` resource configuration matches the label output in the preceding step. The following example queries the `prometheus-example-monitor` ServiceMonitor in the `ns1` project:
```

Comment (eromanova97) [reply modules/monitoring-investigating-why-user-defined-metrics-are-unavailable.adoc]: Fixed, thank you!

File: modules/monitoring-investigating-why-user-defined-metrics-are-unavailable.adoc [hunk 2]
```diff
@@ -39,18 +39,21 @@ $ oc -n ns1 get service prometheus-example-app -o yaml
     app: prometheus-example-app
 ----
 +
-.. Check that the `matchLabels` `app` label in the `ServiceMonitor` resource configuration matches the label output in the preceding step:
+.. Check that the `matchLabels.app` label in the `ServiceMonitor` resource configuration matches the label output in the preceding step. The following example queries the `prometheus-example-monitor` service monitor in the `ns1` project:
```
Comment (simonpasquier) [inline modules/monitoring-investigating-why-user-defined-metrics-are-unavailable.adoc:42]:
```suggestion
.. Check that the `matchLabels` definition in the `ServiceMonitor` resource configuration matches the label output in the preceding step. The following example queries the `prometheus-example-monitor` service monitor in the `ns1` project:
```

**Comment by juzhao** [comment]
LGTM

**Review by adellape — APPROVED** [review]

**Pull request merged**